src/config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "apiUrl": "https://puppyapi.azurewebsites.net/api"
+  "apiUrl": "https://localhost:44399/api"
```

Comment:
Konfiguraciu endpointu, ktory je zavisly na type prostredia by bolo vhodne dat do suboru `.env`. Viac o tom je tu https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env vysvetlit si to mozme na calle

Reply:
Podotazka - mas rozbehany endpoint aj lokalne cez ten projekt v druhom repozitary?

Reply:
toto som tiez zabudol nechcel som to commitovat. a ano mam rozbehane vsetko aj lokalne frontend, bakcend aj databaza